torch/nn/quantized/modules/conv.py

```diff
@@ -0,0 +1,104 @@
+# coding=utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import torch
+from torch.nn.modules.utils import _pair
+from torch.nn.quantized import functional as qF
+
+from torch.nn.modules.conv import _ConvNd
+
+"""Computes the output shape given convolution parameters."""
+def _conv_output_shape(input_size, kernel_size, padding, stride, dilation,
+                       output_padding=0):
+    return np.floor((input_size + 2 * padding - kernel_size - (kernel_size - 1)
+                    * (dilation - 1)) / stride) + 2 * output_padding + 1
+
+
+class Conv2d(_ConvNd):
```

Comment:
can you add `from_float` and serialization that's similar to `nn.quantized.Linear`?
can be in another two prs.